azure-spring-boot-parent/pom.xml

```diff
@@ -34,8 +34,18 @@
         <powermock.junit4.version>1.7.0</powermock.junit4.version>
         <powermock.api.mockito2.version>1.7.0</powermock.api.mockito2.version>
         <commons.logging.version>1.2</commons.logging.version>
+        <azuremonitor.micrometer.registry>1.1.0-SNAPSHOT</azuremonitor.micrometer.registry>
+        <micrometer.core>1.1.0-SNAPSHOT</micrometer.core>
     </properties>
 
+    <repositories>
```

Comment:
This is the location of snapshot repository where the snapshot build for Micrometer is published.